docs/concepts/clients/app.md

```
@@ -0,0 +1,182 @@
+# App
+
+OmAgent App is an app for developers to visualize and edit business scenario content for large model capabilities. It supports interaction with multimodal large models by obtaining data from mobile phone cameras, audio streams, etc., and combining memory, tool invocation, and other capabilities. Based on the business scenarios developed by users, it outputs corresponding content through interaction with the Agent service, providing a demonstration app for intelligent agent scenario development with multimodal content input and output. Next, we will introduce the relevant functions of OmAgent App step by step to start the user's experience journey.
+
+## App Installation
+The QR code for downloading the app is as follows:
+
+<p align="center">
+  <img src="../../images/app_qrcode.png" width="300"/>
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)